### PR TITLE
Fix 1826

### LINKED ISSUE #1826: Custom continents don't work
VCS promises support of custom continents, looking up files in the user's dot directory. Unfortunately, validation was raising errors on the automatic indexing of continents files. I've got a patch inbound for this.

### PR DESCRIPTION
**Pull request by chaosphere2112**
Fixes #1826 and updates the test to properly cover all of the ways that we intake continents arguments.

### CODE REVIEW

**Comment by chaosphere2112** [comment]
Baseline: UV-CDAT/uvcdat-testdata#109

**Comment by chaosphere2112** [comment]
@aashish24 @doutriaux1

**Comment by chaosphere2112** [comment]
@aashish24 @doutriaux1 I was mistaken; this PR is totally ready to go.

**Comment by chaosphere2112** [comment]
@doutriaux1

File: Packages/vcs/Lib/Canvas.py [hunk 1]
```diff
@@ -806,7 +806,7 @@ def _reconstruct_tv(self, arglist, keyargs):
                 contout = 0
 
         if (isinstance(arglist[GRAPHICS_METHOD], str) and (arglist[GRAPHICS_METHOD]) == 'meshfill') or (
-                (xdim >= 0 and ydim >= 0 and (contout >= 1) and (contout < 12))):
+                (xdim >= 0 and ydim >= 0 and (contout >= 1))):
```
Comment (aashish24) [inline Packages/vcs/Lib/Canvas.py]: @chaosphere2112 what was 12 for?

File: testing/vcs/test_continents.py [hunk 2]
```diff
@@ -43,12 +43,25 @@
     cont_line.type = line_styles[i % 5]
     cont_line.color = i + 200
     template = multitemplate.get(i)
-    if cont_index != 3:
+    if cont_index != 3 and i != 4 and i != 11:
```
Comment (aashish24) [inline testing/vcs/test_continents.py]: @chaosphere2112 what 4 and 11 meant?

**Comment by chaosphere2112** [comment]
@doutriaux1 Can we get this in the release?

**Comment by doutriaux1** [comment]
@chaosphere2112 testdata has conflict now... can you please update? Thanks!

**Pull request merged**